blocks/library/latest-posts/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import Placeholder from 'components/placeholder';
+
+/**
+ * Internal dependencies
+ */
+import { registerBlockType } from '../../api';
+import { getLatestPosts } from './data.js';
+
+registerBlockType( 'core/latestposts', {
+	title: wp.i18n.__( 'Latest Posts' ),
```

Comment:
I think yes. However, how can I import `wp.*`? `wp` is a global variable which is already present so I'm not sure how I would import it.